### ISSUE TITLE
[package] gtest/1.1.0: CONAN_PKG::gtest target not found

### ISSUE THREAD

**Issue by dmsurti** [conan-io/conan-center-index#7993]
### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **gtest/1.10.0**
  * Operating System+version: **Linux Ubuntu 20.04**
  * Compiler+version: **GCC 9**
  * Conan version: **conan 1.40.3**

### Details

Linking using the CONAN_PKG::gtest target doesn't seem to work anymore after #7890 got merged

```
target_link_libraries(test PUBLIC CONAN_PKG::ms-gsl)
```
Will produce

```
CMake Error at tests/unit/CMakeLists.txt:15 (add_executable):
  Target "test" links to target "CONAN_PKG::gtest" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?
```

Using the recipe 197d6e9c456cdaf7d8301e2300a8da6b works fine.

This is similar to the issue #7925.

**Comment by SpaceIm** [comment]
It's an issue of set_property() in conan client, it will be fixed in 1.42.1: https://github.com/conan-io/conan/pull/9952

**Comment by Stateford** [comment]
Still having issues on windows for 1.42.1

- Package Name/Version: gtest/1.11.0
- Operating System+version: Windows 10
- Compiler+version: MSVC
- Conan version: conan 1.42.1

**Comment by dmsurti** [comment]
@Stateford Just an FYI: `gtest/1.10.0` works fine now on Windows with conan `1.42.1`

**Comment by Stateford** [comment]
> @Stateford Just an FYI: `gtest/1.10.0` works fine now on Windows with conan `1.42.1`

I can't seem to get gtest/1.10.0 to work with conan 1.42.1 still.

```
Target "test" links to target "CONAN_PKG::gtest" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?
```

**Comment by dmsurti** [comment]
@Stateford You may want to try delete your local cache or point your cache to a new location. https://docs.conan.io/en/latest/mastering/custom_cache.html

**Comment by SpaceIm** [comment]
Are you sure to have called `conan install` again such that `conanbuildinfo.cmake` is updated? Same than others, it works fine on my machine with conan 1.42.1

**Comment by Stateford** [comment]
@dmsurti @SpaceIm 
I have tried cleaning the cache and it would not work still.

So I'm not sure if this is causing the issue, but I'm using conan-cmake.

```
conan_cmake_run(
    REQUIRES gtest/1.10.0
    BASIC_SETUP CMAKE_TARGETS
    BUILD missing
)
...
    target_link_libraries(${TEST_NAME} CONAN_PKG::gtest)
```

The issue seemed to be resolved on linux.

I did find a work-around for now using a generator and find_package, but it was nice being able to just use it this way.

**Comment by SpaceIm** [comment]
Did it work with CMake reconfiguration?

**Comment by Stateford** [comment]
I've tried and it still won't work. It's also failing on CI/CD where we have it running on a clean system every run.

**Comment by wbitesi** [comment]
@Stateford I had the problem with a similar library too. 1.42.1 fixed the linux build and 1.42.2 fixed the windows build. conan-cmake uses cmake_multi generator by default. (Fix in https://github.com/conan-io/conan/pull/10062)

**Comment by perseoGI** [comment]
Hi there, 
I was reviewing old issues and I have come across this one.
The `gtest/1.1.0` version is no longer supported by Conan Center Index.

I'm closing this issue as `CONAN_PKG` is `conan v1` related and thus, we are avoiding giving support to old versions.
If for some reason the problem remains nowadays, do not hesitate to create a new ticket with updated information, preferably using `conan v2`.

Happy coding 🐸
